### PR TITLE
[OPS] It would be nice if the builder did not have a tiny (or any) PH…

### PR DESCRIPTION
**Pull request by cafuego**
…P memory_limit, so icnlude the same file we use in the php7 base image.

### CODE REVIEW

**Comment by teodorescuserban** [comment]
Happy to review once you add me as reviewer.

File: alpine-php/builder/Dockerfile.tmpl [hunk 1]
```diff
@@ -56,6 +57,7 @@ RUN npm install --global npm@latest && \
       gzip \
       openssh-client \
       patch \
+      python3 \
```
Comment (cafuego) [inline alpine-php/builder/Dockerfile.tmpl:60]: Cool story: HPC Viewer theme asset generation hard requires python2.

Comment (teodorescuserban) [reply alpine-php/builder/Dockerfile.tmpl:60]: warning, from alpine 3.12, the python 2 package has been renamed from `python` to `python2`

**Review by teodorescuserban — APPROVED** [review]

File: alpine-php/builder/Dockerfile.tmpl (hunk 1 above)
Comment (cafuego) [reply alpine-php/builder/Dockerfile.tmpl:60]: It can go and update its workflow. We don't support python2.

**Pull request merged**